exercises11/main.js

```diff
@@ -0,0 +1,124 @@
+// exercise 1: Viết 1 hàm trả về 1 array, nhận vào 2 tham số, tham số thứ 1 là ký tự cần lặp, tham số thứ 2 là số lần lặp
+//cách 1
+function returnArray (charRepeat, numberRepeat) {
+  if (typeof numberRepeat === 'number' && numberRepeat > 0) {
+    const array = []
+    for (let i = 1; i <= numberRepeat; i ++) {
+      array.push(charRepeat)
+    }
+    return array
+  }
+  return 'số lần lặp phải là kiểu số và lớn hơn 0'
+}
+
+// exercise 2
+function reverseArray (array) {
+  if (Array.isArray(array)) {
+    for (let i = 0; i < array.length/2; i ++) {
+      const provisional = array[i]
+      array[i] = array[array.length-1-i]
+      array[array.length-1-i] = provisional
```

Comment:
Cái gì rối quá
Sao ko for ngược từ cuối đến đầu
Mỗi lần lặp thì push item vào mảng mới